pipeline/scripts/zip-mac-folder.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const child_process = require('child_process');
```

Comment:
Here too, would be nice to have a comment describing why this is necessary/referencing appropriate github issue